LICENSES-THIRD-PARTY.TXT

```diff
@@ -18,6 +18,7 @@ The following third party software components are governed under both Apache 2.0
   - lazy_static, (c) 2016 lazy-static.rs Developers https://github.com/rust-lang-nursery/lazy-static.rs
   - log, (c) 2015 The Rust Project Developers https://github.com/rust-lang/log
   - serde_json, (c) 2017 Serde Developers https://github.com/serde-rs/json
+  - auto-value-gson extension, (c) 2015 Ryan Harter https://github.com/rharter/auto-value-gson
```

Comment:
If it is a build-time dependency, it must not be here (= we will not disribute it).